### DISCUSSION TITLE
Look up window by position isn't handy

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#1987] [Q&A]
The old position tab deserves an improvement. When introducing the coordinates by the first time, it goes there, but if Stellarium decides to take you somewhere else on its own (and yes, it does) and you need to recenter in the coordinates, you will find no button to do so. Instead, as a workaround, you may change some digit, press enter, go there and then enter again the right digit to come back to the right place. Annoying. Can that be fixed, please?

**Comment by gzotti** [comment]
Can you please describe this more clearly? "Old position tab" in which dialog? Usually you can recenter a selected object by pressing space. Or is this a problem with driving telescopes?

**Comment by axd1967** [comment]
This could be the culprit:

![image](https://user-images.githubusercontent.com/3529789/138701268-7274320c-e9cf-43b6-8c1a-4cb89bde330d.png)

The input boxes remember the last entered position, but a button is missing to go back to that position (e.g. after slewing around, selecting an object, ...). The workaround is to change the same (in this case RA or DEC) digit twice, as each change re-positions the direction and the second change re-centers on the previously selected value.

Another small issue is that changing the coordinate system resets the values - which is painful. Better would be to recompute the existing coordinates into the newly selected coordinate system and update the values accordingly. The idea is always: protect the user from losing data.

Note that this box could benefit from a new button: `Current position` (that will fill in the current position in the input boxes.)
